data/parsers/france.py

```diff
@@ -63,5 +63,7 @@ def parse():
     regions2 = {}
     for reg, d in regions.items():
         regions2['-'.join(['FRA',reg])] = [d[day] for day in sorted(d.keys())]
+        
+    regions2 = add_cases(regions2, list(regions2.keys()), 'France', cols)
```

Comment:
We get France country data from ECDC parser currently. Did you compare the values? I will have to look into what happens if we have the same country listed twice (I think it depends on the alphabetic order of folders, so right now the ecdc numbers might take precedence in the .json generation).

Reply:
You are missing the add_cases import in this file as well

Reply:
I looked into the the merging question now. As I predicted, the ECDC data takes precedence over the france parser currently. That being said, ECDC does not provide hospital and ICU data, so this PR is definitely valueable, as we fill missing ECDC data with output of this parser. As a note, france parser's death count is considerably lower than the ECDC data, and we currently use the ECDC data instead.

Reply:
Thanks for looking into this and figuring everything out. The reason for the ECDC death data being considerably higher is most probably because it is considering overall Covid deaths while the regional source of the parser only hospital death. Most valuable is indead to have the values of ICU and hospitalized for France as a whole by summing the regional data. Thanks again.